### PR TITLE
[buffer] add spheroid to buffer strategies as an optional constructor parameter

### PR DESCRIPTION
**Pull request by barendgehrels**

### CODE REVIEW

**Comment by vissarion** [comment]
> All checks fail because of
> 
> ```
> W: GPG error: http://dk.archive.ubuntu.com/ubuntu xenial InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 40976EAF437D05B5 NO_PUBKEY 3B4FE6ACC0B21F32
> ```
> 
> Is that at my side (might be) or is there something changed in the CI? @awulkiew @mloskot ?

I think this is general (for `master` too). See https://github.com/boostorg/geometry/pull/1102

**Comment by barendgehrels** [comment]
> > All checks fail because of
> > ```
> > W: GPG error: http://dk.archive.ubuntu.com/ubuntu xenial InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 40976EAF437D05B5 NO_PUBKEY 3B4FE6ACC0B21F32
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Is that at my side (might be) or is there something changed in the CI? @awulkiew @mloskot ?
> 
> I think this is general (for `master` too). See #1102

Thanks!

File: test/algorithms/buffer/buffer_geo_spheroid.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+namespace
+{
+    int const points_per_circle = 360;
+    std::string const road = "LINESTRING(10.3966569 63.4276957,10.3998059 63.4279182,10.4000859 63.4283889,10.3982915 63.4284015,10.3980902 63.4288772,10.3987772 63.4288520)";
+    std::string const torg = "POINT(10.3937759 63.4302323)";
+}
+
+template <typename Formula, bool Clockwise, typename Point, typename Spheroid>
+void test_linestring(std::string const& label, Spheroid const& spheroid,
+    double expected_area_round, double expected_area_miter)
```
Comment (vissarion) [inline test/algorithms/buffer/buffer_geo_spheroid.cpp:21]: should this be aligned under `(`?

**Review by vissarion — APPROVED** [review]
Thanks!

File: test/algorithms/buffer/buffer_geo_spheroid.cpp (hunk 1 above)
Comment (barendgehrels) [reply test/algorithms/buffer/buffer_geo_spheroid.cpp]: :heavy_check_mark:

**Pull request merged**